src/hooks/screen/useChatScreenHook.ts

```diff
@@ -47,6 +63,14 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     }
   };
 
+  const handleScrollTo = (messageId: string) => {
+    const index = chats.findIndex((message) => message?.id === messageId);
+    if (index !== -1) {
+      flatListRef.current?.scrollToIndex({index});
+      setSelectedMessageId(messageId);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `handleScrollTo` is introduced which scrolls to a specific message in the chat. This seems to be a useful feature for navigating through the chat history.
```diff
+ const handleScrollTo = (messageId: string) => {
+     const index = chats.findIndex((message) => message?.id === messageId);
+     if (index !== -1) {
+       flatListRef.current?.scrollToIndex({index});
+       setSelectedMessageId(messageId);
+     }
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->